susemanager-utils/testing/docker/head/uyuni-head-base/add_repositories.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+zypper rr 'NON OSS'
+zypper rr 'NON OSS Update'
+
+zypper ar -f http://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master/openSUSE_Leap_42.3/ "Uyuni:Master:42.3"
+zypper ar -f http://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/Other/openSUSE_Leap_42.3/ "Uyuni:Master:Other:42.3"
+#zypper ar -f http://download.suse.de/ibs/Devel:/Galaxy:/pylint/SLE_12_SP3/ "pylint"
```

Comment:
Shouldn't we just remove this line?

Reply:
Not sure yet. This is a reminder to check if we need such an extra repo for Uyuni as well.
The main reason might be to have a dedicated pylint version because every version checks different.